README.md

```diff
@@ -27,7 +27,7 @@ FastStore is an open ecommerce framework built using modern technologies that ai
 - Jamstack based (deploy with Netlify, Vercel and any other JAMStack platform)
 - Marketplace ready with support to millions of skus
 - Intelligence via Google Analytics and GTM
-- Evergreen stores with our no-fork based approach
+- Evergreen stores with auto updates
```

Comment:
How about "Evergreen stores that are always updated" or even "Evergreen stores that are automatically updated"?